xml/bp_troubleshooting_report.xml

```diff
@@ -27,14 +27,41 @@
   </para>
 <screen>&prompt.user;rpm -q supportutils-plugin-ses</screen>
   <para>
-   If it is missing on the &ceph; server, install it with
+   If it is missing on the &ceph; server, install it with:
   </para>
 <screen>&prompt.root;zypper ref &amp;&amp; zypper in supportutils-plugin-ses</screen>
+  <para>
+    <literal>supportutils-plugin-ses</literal> will try to capture as much
+    useful information as it can about both the &ceph; cluster as a whole,
+    and about the host on which it is running. Some information, including
+    <command>ceph daemon status</command>, <command>podman</command> image information, <command>rpm</command> verification and some
```

Comment:
I don't think "ceph daemon status" should be a command here -- it's just meant to mean the status of all the ceph daemons in this context.